packages/cli/src/commands/generate/script/script.js

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs'
+import path from 'path'
+
+import Listr from 'listr'
+import { paramCase } from 'param-case'
+import terminalLink from 'terminal-link'
+
+import { getPaths, writeFilesTask } from 'src/lib'
+import c from 'src/lib/colors'
+
+const POST_RUN_INSTRUCTIONS = `Next steps...\n\n   ${c.warning(
+  'After writing your script, you can run it with:'
+)}
+
+     yarn rw run <name>
```

Comment:
This should be exec now right?
```suggestion
     yarn rw exec <name>
```